core/templates/components/state-editor/state-editor.directive.ts

```diff
@@ -121,6 +121,28 @@ angular.module('oppia').directive('stateEditor', [
             $scope.conceptCardIsShown = !$scope.conceptCardIsShown;
           };
 
+          var questionSuggestionReview = angular.element(
+            document.querySelector('.oppia-question-suggestion-review'));
+          questionSuggestionReview.on('scroll', function() {
+            const stickyHeaderScrollTop = 50;
+            var tutorialStateContent = document.getElementById(
+              'tutorialStateContent');
+            var cardAvatarOnSticky = document.querySelector(
+              '.oppia-editor-card-avatar');
+            if (questionSuggestionReview.scrollTop() >
+              stickyHeaderScrollTop) {
+              tutorialStateContent.classList.remove(
+                'state-content-editor-parent-container');
+              cardAvatarOnSticky.classList.add(
+                'oppia-editor-card-avatar-sticky');
+            } else {
+              tutorialStateContent.classList.add(
+                'state-content-editor-parent-container');
+              cardAvatarOnSticky.classList.remove(
+                'oppia-editor-card-avatar-sticky');
```

Comment:
How important is it that we show a padding before scroll? I'm wondering, in the interest of saving space in the question review modal, can we remove padding in the state content irrespective of the scroll position? This would mean a compact view for content when reviewing questions by default.